app/src/main/java/ch/epfl/qedit/model/Server.java

```diff
@@ -0,0 +1,115 @@
+package ch.epfl.qedit.model;
+
+import androidx.annotation.NonNull;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class Server {
+    private boolean isSignInSuccessful = false;
+    private boolean isCreateSuccessful = false;
+    private FirebaseAuth firebaseAuth = null;
```

Comment:
Looking at the documentation, it seems that we should use `FirebaseAuth.getInstance()` to get a new instance of the authentication system.

Reply:
That is exactly what happens in `initServer()`

Reply:
Maybe I'm wrong but look here:
https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuth

Reply:
My bad, I didn't see ;)